src/app/models/email.ts

```diff
@@ -1,4 +0,0 @@
-export class Email {
```

Comment:
I might be nice to keep this as a type instead of storing them as a string. This may help with validation. Though it is not necessary.